src/CMakeLists.txt

```diff
@@ -119,3 +119,4 @@ add_subdirectory(ts_catalog)
 add_subdirectory(import)
 add_subdirectory(nodes)
 add_subdirectory(planner)
+add_subdirectory(utils)
```

Comment:
in the [current PR there is no](https://github.com/timescale/timescaledb/actions/runs/5410151601/jobs/9831219860?pr=5832#step:4:186) `CMakeLists.txt` in that directory

Reply:
@nikkhils it would be great if you would be able to upload that file

Reply:
@nikkhils could you please upload the missing file?